discovery-provider/scripts/start.sh

```diff
@@ -81,10 +81,10 @@ if [ -z "$audius_db_url" ]; then
         sudo -u postgres pg_ctl init -D /db
         echo "host all all 0.0.0.0/0 md5" >>/db/pg_hba.conf
         echo "listen_addresses = '*'" >>/db/postgresql.conf
-        sudo -u postgres pg_ctl start -D /db
+        sudo -u postgres pg_ctl start -D /db -o "-c shared_buffers=512MB" -o "-c shared_preload_libraries=pg_stat_statements"
         sudo -u postgres createdb audius_discovery
     else
-        sudo -u postgres pg_ctl start -D /db
+        sudo -u postgres pg_ctl start -D /db -o "-c shared_buffers=512MB" -o "-c shared_preload_libraries=pg_stat_statements"
```

Comment:
thanks, this is great! although long-term i still do believe in the db being similarly containerized being a helpful thing.

I know you were trying to save complexity in number of containers running, but maybe if we are better about port mgmt & naming of containers, we'll make life a lot easier

discovery_1_indexer
discovery_1_server
discovery_1_db

discovery_2_indexer
etc. etc.

Reply:
Yes, this sounds good, I've made a follow up task for this, thanks!